buildkite/scripts/cache-through.sh

```diff
@@ -24,16 +24,18 @@ fi
 UPLOAD_BIN=/usr/local/google-cloud-sdk/bin/gsutil
 PREFIX=gs://buildkite_k8s/coda/shared
 FILE="$1"
-CMD="$2"
+HIT_CMD="$2"
+MISS_CMD="$3"
 
 set +e
 if ! $UPLOAD_BIN cp ${PREFIX}/${FILE} .; then
```

Comment:
Worth checking if the file exists locally beforehand (since the path only depends on the agent name and organization and so there should technically be local cache hits following a single miss)?

For example, `/var/buildkite/builds/gke-east-buildkite-agent-77bc94c7cc-qjr5d-1/o-1-labs-2/coda/...` would be the download DIR.

Reply:
I think buildkite wipes the working dir each build but we could save the data outside the work dir. Though we can do that in a followup! For now, I did add a `-f` check for the file.